src/app/dialogs/dialog-loan-lend/dialog-loan-lend.component.scss

```diff
@@ -1,397 +1,124 @@
+.wrapper-icons {
+  width: 35px;
   height: 35px;
   display: flex;
   align-items: center;
-}
-
-.--data-margin {
-  margin-left: 10px;
-}
-
-.item {
-    margin-top: 8px;
-    margin-bottom: 8px;
-}
-
-.currency {
-    font-size: 18px;
-    font-family: 'proxima-nova', sans-serif;
-    font-weight: 500;
-}
-
-.exchange {
-  background-color: none;
-  height: auto;
-  display: flex;
-  align-items: center;
-  .rate {
-    font-size: 12px;
-    color: #6D6E6F;
-  }
-  i {
-    cursor: pointer;
-    font-size: 1.5em;
-  }
-}
-
-img {
+  margin: 8px 20px;
+  border-radius: 50px;
+  background-color: white;
+  padding: 3px;
+  img {
+    display: inline-block;
     max-height: 100%;
     margin: auto;
-}
-
-.data {
-    font-size: 14px;
-    font-family: 'Roboto', sans-serif;
-    font-weight: 800;
-    color: #ADADAD;
-    display: flex;
-    flex-wrap: nowrap;
-}
-
-.inline {
-    display: inline-block;
-}
-
-.--auto {
-  width: auto !important;
-}
-
-.in {
-    display: inline;
-}
-
-.img-container{
-  text-align: center;
-  margin-top: 20px;
-  img{
-    height: 150px;
-    margin: 0 auto;
   }
 }
 
-.wrap {
-  display: flex;
-  flex-wrap: nowrap;
-}
-
-.title {
-  font-size: 14px;
-}
-.account {
-  display: none;
-  font-family: 'Roboto', sans-serif;
-  color: #ADADAD;
-}
-
-.mobile {
-  display: block;
-}
-
-.desktop {
-  display: none;
-}
-
-.cancel {
-  display: none;
-}
-
-.title-2 {
-  font-size: 16px;
-  font-weight: 500;
-}
-
-.no-gutters {
-  margin-right: 0;
-  margin-left: 0;
-  padding: 0;
-
-  > .col,
-  > [class*="col-"] {
-    padding-right: 0;
-    padding-left: 0;
+.accordion-details {
+  mat-expansion-panel {
+    margin-top: 35px;
   }
 }
 
-.wrap {
-  display: flex;
-  flex-wrap: nowrap;
-}
-
-.no-margin {
-  margin: 0;
-}
-
-.vertical-center {
-  display: flex;
-  align-items: center;
-}
-
-.baseline {
-  display: flex;
-  align-items: center;
-}
-
-.fa-arrow-right {
-  margin-left: 10px;
-}
-
-.margin-left {
-  margin-left: 5px;
-}
-
-.cero-box {
-  height: 40px;
-  display: flex;
-  align-content: flex-end;
-}
-
-.mat-raised-button {
-  width: 100%;
-}
-.pad-left {
-  padding-left: 0;
-}
-
-.fa-arrow-left {
-  color: white;
-}
-
-.arrow-space {
-  display: none;
-}
-
-.cur {
-  font-size: 14px;
-  color: white;
-}
-
-.center-data {
-  display: flex;
-  align-items: center;
-}
-
-.check {
-  font-size: 16px;
-  color: #6EB548;
-}
-
-.triangle {
-  font-size: 16px;
-  color: #EAA219;
-  cursor: pointer;
-}
-
-.end {
-  display: flex;
-  align-items: baseline;
-}
-
-.--lend-amount,
-.--lend-currency,
-.--return-amount,
-.--return-currency {
-  @include styled-font('Roboto', 700, 16px);
-}
-
-@media (min-width: 768px){
-  .dialog-content {
-    .accordion-details {
-      mat-expansion-panel {
-        margin-top: 60px;
+@media (min-width: 768px) {
+  .dialog {
+    max-width: 800px;
+    .dialog-content {
+      .item {
+        &__text-currency {
+          &--title {
+            @include styled-font("Roboto", 700, 26px);
```

Comment:
@martinbobbio Use typography-headline(4) instead of custom styled-font

Reply:
Implemented